front/public/locales/fr/stdcm.json

```
@@ -156,24 +156,34 @@
         "errorMissingConsist": "Données de convoi manquantes",
         "errorRailwayManager": "Erreur lors de la soumission de la demande à GESICO",
         "followUpMessage": "Vous pourrez assurer le suivi de cette demande dans GESICO.",
+        "for": "Pour",
+        "from": "Du",
         "gauge": "gabarit",
+        "hazardousMaterials": "Marchandises dangereuses",
         "maxSpeed": "vitesse max.",
         "newRailManagerRequest": "Nouvelle demande GESICO",
+        "noCategory": "Sans catégorie",
         "other": "Autres parcours",
         "pathType": "Type de parcours",
         "pdf": "PDF",
         "referenceEngine": "engin de référence",
         "requestedRoute": "PARCOURS DEMANDÉ",
         "retainedSimulation": "SIMULATION RETENUE",
         "rollingStock": "matériel remorqué",
-        "selectPathType": "-- Choisir --",
+        "selectPathType": "Choisir",
         "stop": "Arrêt de {{minutes}} à {{stopAt}}",
         "successMessage": "Demande reçue par GESICO",
         "toleranceAfter": "Tolérance aval",
         "toleranceBefore": "Tolérance amont",
         "totalLength": "longueur totale",
         "totalTonnage": "tonnage total",
-        "verifyParameters": "Voici les paramètres de requête et la simulation qui seront transmis à GESICO, prenez le temps de les vérifier."
+        "trainCategory": "Catégorie de circulation",
+        "trainToBeSubstituted": "Sillon à substituer",
+        "verifyParameters": "Voici les paramètres de requête et la simulation qui seront transmis à GESICO, prenez le temps de les vérifier.",
+        "warnings": {
+          "hazardousMaterialsBold": "Les restrictions des convois de matières dangereuses ne sont pas pris en compte par l’algorithme LMR.",
+          "hazardousMaterialsNormal": "La simulation sera vérifiée par un horairiste lors du traitement du sillon, et pourrait être irrecevable."
```

Has this part changed compared to the mockup ?
```suggestion
          "hazardousMaterialsNormal": "La faisabilité de la demande sera étudiée par un horairiste lors du traitement du sillon, et pourrait être irrecevable."
```